browser/ui/webui/brave_settings_ui.cc

```diff
@@ -169,6 +179,13 @@ void BraveSettingsUI::AddResources(content::WebUIDataSource* html_source,
       "verticalTabStripFeatureEnabled",
       base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs));
 #endif
+
+#if BUILDFLAG(ENABLE_AI_CHAT)
+  html_source->AddBoolean("isLeoAssistantDisabled",
+                          !ai_chat::features::IsAIChatEnabled());
+#else
+  html_source->AddBoolean("isLeoAssistantDisabled",true);
```

Comment:
nit: prefer to include the substring "feature" here so it's clear this is about the feature being disabled as a flag vs user-enabled as a pref

Reply:
changed the flag name to `isLeoAssistantAllowed`
https://github.com/brave/brave-core/pull/19212/commits/8b61de993579fa833a3b6bb349c5d4ee481e152c